_docs/tasks/traffic-management/circuit-breaking.md

```diff
@@ -0,0 +1,172 @@
+
+## Before you begin
+
+* Setup Istio by following the instructions in the
+  [Installation guide]({{home}}/docs/setup/).
+  
+* Start the [httpbin](https://github.com/istio/istio/tree/master/samples/httpbin) sample
+  which will be used as the backend service for our task
+  
+  ```bash
+  kubectl apply -f <(istioctl kube-inject -f samples/httpbin/httpbin.yaml)
+  ```  
+    
+
+
+## Circuit breaker
+
+Let's set up a scenario to demonstrate the circuit-breaking capabilities of Istio. We should have the `httpbin` service running from the previous section. We'll want to make sure we set some route rules to route all traffic to `httpbin` with `version=v1`. We'll be setting the circuit-breaking settings by creating an Istio [destination policy]({{home}}/docs/reference/config/traffic-rules/destination-policies.html) but for destination policies to take effect, there needs to be a route rule for the destination first. Let's create the route rule:
+
+### Creating circuit breaking policies
+
+1. Create a default route rule to route all traffic to `v1` of our `httpbin` service:
+
+```bash
+istioctl create -f samples/httpbin/routerules/httpbin-v1.yaml
+```
+
+2. Create a [destination policy]() to specify our circuit breaking settings when calling `httpbin` service:
+
+   ```bash
+   cat <<EOF | istioctl create -f -
+   apiVersion: config.istio.io/v1beta1
+   kind: DestinationPolicy
+   metadata:
+     name: httpbin-circuit-breaker
+   spec:
+     destination:
+       name: httpbin
+       labels:
+         version: v1
+     circuitBreaker:
+       simpleCb:
+         maxConnections: 1
+         httpMaxPendingRequests: 1
+         sleepWindow: 3m
+         httpDetectionInterval: 1s
+         httpMaxEjectionPercent: 100
+         httpConsecutiveErrors: 1
+         httpMaxRequestsPerConnection: 1
+   EOF
+   ```
+   
+3. Verify our destination policy was created correctly:
+
+    ```bash
+    istioctl get destinationpolicy
+    
+    NAME                    KIND                                            NAMESPACE
+    httpbin-circuit-breaker DestinationPolicy.v1alpha2.config.istio.io      istio-samples
+    ```   
+    
+### Setting up our client
+
+Now that we've set up rules for calling the `httpbin` service, let's create a client we can use to send traffic to our service and see whether we can trip the circuit breaking policies. We're going to use a simple go client called [gobench](https://github.com/cmpxchg16/gobench). With this client we can control the number of connections, concurrency, and delays of outgoing HTTP calls. In this step, we'll set up a client that is injected with the istio sidecar proxy so our network interactions are governed by Istio:
+
+    ```bash
+    kubectl apply -f <(istioctl kube-inject -f samples/httpbin/sample-client/gobench-deploy.yaml)
```

Comment:
As we chatted, let's use fortio instead (could also change httpbin btw but at least the driver)

Let me know if I can help with the setup/doc/...